packages/reakit-playground/src/PlaygroundPreview.tsx

```diff
@@ -84,14 +84,18 @@ export function PlaygroundPreview({
           options.modules,
           options.componentName
         );
-        unmount();
+        
         ReactDOM.render(renderChildren(<Example />), ref.current);
       } catch (e) {
         unmount();
```

Comment:
Maybe removing it fixes the issue?
```suggestion
```

Reply:
Actually, I think I'd try to keep this `useEffect` closure intact and create a separate `useEffect` with no deps:
```js
React.useEffect(() => unmount, []);
```

Reply:
Sure - That would also work. 
I've done this change directly on Github, and haven't tested it locally. 🤞